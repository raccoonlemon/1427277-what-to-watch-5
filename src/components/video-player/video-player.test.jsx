import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import VideoPlayer from "./video-player";

const renderer = new ShallowRenderer();

it(`<VideoPlayer> renders correctly`, () => {
  renderer.render(<VideoPlayer poster={`123`} src={`123`}/>);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
