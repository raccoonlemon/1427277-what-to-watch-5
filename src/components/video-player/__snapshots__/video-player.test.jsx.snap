// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer> renders correctly 1`] = `
<React.Fragment>
  <video
    height="175"
    muted="muted"
    poster="123"
    width="280"
  >
    <source
      src="123"
    />
  </video>
</React.Fragment>
`;
